.github/workflows/wheel_linux_x86_64.yml

```diff
@@ -76,8 +76,7 @@ jobs:
             source /opt/rh/devtoolset-11/enable -y
 
           CIBW_ENVIRONMENT: |
-            PATH=/opt/rh/devtoolset-11/root/usr/bin:$PATH
-            BACKEND=${{ matrix.exec_model }}
+            PATH=/opt/rh/devtoolset-11/root/usr/bin:$PATH BACKEND=${{ matrix.exec_model }}
```

Comment:
Hi @multiphaseCFD, could you please explain why you needed to make this change?

Reply:
Hi @AmintorDusko , we need to separate the assignments by spaces to specify more than one environment variable.